_targets/build.project.armv7a9-zynq7000

```diff
@@ -99,7 +99,8 @@ b_build_target() {
 b_image_target() {
 	b_mkscript_user "${USER_SCRIPT[@]}"
 	b_prod_image
-	b_dev_image
+	# FIXME: DEV_USER_SCRIPT is unset here - can't build dev image
```

Comment:
@HBuczynski: this is just a quickfix to make zedboard compilable, please verify if dev image should be built here